spi/src/index.ts

```diff
@@ -49,6 +49,9 @@ async function main() {
     // STEP 6. Transforms SPI Design Components into SIMS Sampling Sites
     await transformSampleSites(connection);
```

Comment:
It looks like the same function is being called twice

Reply:
fixed! and added other functions from the remaining completed transformations in